driver/SDL/SDL_keyboard.h

```diff
@@ -0,0 +1,73 @@
+/**
+ * @file keyboard.h
+ *
+ */
+
+#ifndef KEYBOARD_H
+#define KEYBOARD_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*********************
+ *      INCLUDES
+ *********************/
+#ifdef LV_CONF_INCLUDE_SIMPLE
+#include "lv_drv_conf.h"
+#else
+#include "lv_drv_conf.h"
+#endif
+
+#if USE_KEYBOARD
+#include <stdint.h>
+#include <stdbool.h>
+#include <SDL2/SDL.h>
```

Comment:
`SDL2/SDL.h` is already included below, with `#include MONITOR_SDL_INCLUDE_PATH`.